tests/console/validate_repos.pm

```diff
@@ -14,6 +14,7 @@ use scheduler 'get_test_suite_data';
 
 sub run {
     my $test_data = get_test_suite_data();
+    my %expected_repos = map { $_->{alias} => 1 } @{$test_data->{repos}};
```

Comment:
below this you need to get @actual_aliases from zypper -n lr --uri some parsing.